torchvision/models/detection/ssd.py

```diff
@@ -0,0 +1,486 @@
+import torch
+import torch.nn.functional as F
+import warnings
+
+from collections import OrderedDict
+from torch import nn, Tensor
+from typing import Any, Dict, List, Optional, Tuple
+
+from . import _utils as det_utils
+from .anchor_utils import DBoxGenerator
+from .backbone_utils import _validate_trainable_layers
+from .transform import GeneralizedRCNNTransform
+from .. import vgg
+from ..utils import load_state_dict_from_url
+from ...ops import boxes as box_ops
+
+__all__ = ['SSD', 'ssd300_vgg16']
+
+model_urls = {
+    'ssd300_vgg16_coco': None,  # TODO: Add url with weights
```

Comment:
Originally I had an experimental implementation with ResNet50 backbone. I [removed](https://github.com/pytorch/vision/pull/3403/commits/38e6e7244fd6cf66f344cfefe0e3ec15068dbd9e) it for now and I propose adding more backbones on separate PRs.